app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvUploadForm
+  include ActiveModel::Model
+
+  attr_accessor :keywords
+
+  validates :file, presence: true, csv_keyword_file: true
+
+  def save(file)
+    @file = file
+    return false unless valid?
+
+    entries = keywords.map { |keyword| { content: keyword } }
+    Keyword.create(entries).pluck('id')
```

Comment:
If one of the keyword failed to be created, will the whole transaction get rollback? Also after checking the `create with entries` also `inserted one-at-a-time into the database.` it mean it will be hard for you to trigger the background job later, am I right?

Reply:
for me an easy case for now is just creating multiple keyword, that is an easier solution

```
ActiveRecord::Base.transaction do
  keywords.each do |keyword|
    Keyword.create(content: keyword)
  end
end
```

Reply:
My bad. I was suspicious about whether Keyword.create uses transactions or not. I inspected the log from the test output and saw a transaction block. But turns out, the one who performed the transaction is Rspec, not ActiveModel as I assumed 🥶 
![Screenshot 2023-08-08 at 10 32 03](https://github.com/Goose97/google-scraper-ruby/assets/35915460/3dd8b31f-cf43-43aa-8580-e57298c587ea)

Anw, 333e81a2617b5688c2afa2b0b65a4ac6fe04f200 this